setup/delivery_carrier_category/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
I think that such file in ``.eggs`` folder, should not be added on github. Don't you think ?